packages/common/src/core/schemas/hubproject.ts

```diff
@@ -118,39 +129,58 @@ export const HubProjectEditUiSchema: IUiSchema = {
   type: "Layout",
   elements: [
     {
-      labelKey: "{{i18nScope}}.name.label",
-      scope: "/properties/name",
-      type: "Control",
-    },
-    {
-      labelKey: "{{i18nScope}}.summary.label",
-      scope: "/properties/summary",
-      type: "Control",
-      options: {
-        helperText: {
-          labelKey: "{{i18nScope}}.summary.helperText",
+      type: "Section",
+      options: { headerTag: "h4" },
+      labelKey: "{{i18nScope}}.basicInfo.label",
+      elements: [
+        {
+          labelKey: "{{i18nScope}}.name.label",
+          scope: "/properties/name",
+          type: "Control",
         },
-      },
-    },
-    {
-      labelKey: "{{i18nScope}}.description.label",
-      scope: "/properties/description",
-      type: "Control",
-      options: {
-        helperText: {
-          labelKey: "{{i18nScope}}.description.helperText",
+        {
+          labelKey: "{{i18nScope}}.summary.label",
+          scope: "/properties/summary",
+          type: "Control",
+          options: {
+            helperText: {
+              labelKey: "{{i18nScope}}.summary.helperText",
+            },
+          },
+        },
+        {
+          labelKey: "{{i18nScope}}.description.label",
+          scope: "/properties/description",
+          type: "Control",
+          options: {
+            helperText: {
+              labelKey: "{{i18nScope}}.description.helperText",
+            },
+          },
         },
-      },
+        {
+          labelKey: "{{i18nScope}}.status.label",
+          scope: "/properties/status",
+          type: "Control",
+          options: {
+            enum: {
+              i18nScope: "{{i18nScope}}.status.enum",
+            },
+          },
+        },
+      ],
     },
     {
-      labelKey: "{{i18nScope}}.status.label",
-      scope: "/properties/status",
-      type: "Control",
-      options: {
-        enum: {
-          i18nScope: "{{i18nScope}}.status.enum",
+      type: "Section",
+      options: { headerTag: "h4" },
+      labelKey: "{{i18nScope}}.location.label",
+      elements: [
```

Comment:
Looking at the AC of your issue, it sounds like there's supposed to be another (toggle) field here to optionally hide the map on a project page. Is that still relevant? Or has that been removed from the criteria here?

Reply:
talked to Ben - he'll follow up on this. Getting this merged in for now so others can keep iterating